ddl/ddl.go

```diff
@@ -227,9 +229,12 @@ func newDDL(ctx goctx.Context, etcdCli *clientv3.Client, store kv.Storage,
 	id := uuid.NewV4().String()
 	ctx, cancelFunc := goctx.WithCancel(ctx)
 	worker := &worker{
-		ddlID:      id,
-		etcdClient: etcdCli,
-		cancel:     cancelFunc,
+		schemaVersionSyncer: &schemaVersionSyncer{
+			etcdCli:           etcdCli,
+			selfSchemaVerPath: fmt.Sprintf("%s/%s", ddlSelfSchemaVersion, id),
+		},
+		ddlID:  id,
+		cancel: cancelFunc,
```

Comment:
Do we need it?
By the way, context exist when something is used, not when something is created.
If a object is bind with a context when created, cancel may make things break.